examples/ChatQnA/deploy/gaudi.md

```diff
@@ -87,24 +87,25 @@ ChatQnA megaservice, and UI (conversational React UI is optional). In total,
 there are 8 required and an optional docker images.
 
 ### Build/Pull Microservice images
-::::{tab-set}
-:::{tab-item} Pull
+
+::::::{tab-set}
+
+:::::{tab-item} Pull
 :sync: Pull
 
-To pull pre-built docker images on Docker Hub, proceed to the next step. To customize 
-your application, you can choose to build individual docker images for the microservices 
-before proceeding.
-:::
-:::{tab-item} Build
+If you decide to pull the docker containers and not build them locally,
+you can proceed to the next step where all the necessary containers will
+be pulled in from dockerhub.
+
+:::::
+:::::{tab-item} Build
 :sync: Build
 
 From within the `GenAIComps` folder, checkout the release tag.
 ```
 cd GenAIComps

```

Comment:
Have they cloned the repo in some earlier step?

Reply:
Yes, lines 53-54 is where they clone both GenAIComps and GenAIExamples